test/e2e/examples_test.go

```diff
@@ -65,11 +66,11 @@ func TestSuperTuxKartGameServerReady(t *testing.T) {
 	assert.Equal(t, agonesv1.GameServerStateReady, readyGs.Status.State)
 }
 
-func TestRustGameServerReady(t *testing.T) {
+func TestCppSimpleGameServerReady(t *testing.T) {
 	t.Parallel()
 	gs := &agonesv1.GameServer{
 		ObjectMeta: metav1.ObjectMeta{
-			GenerateName: "rust-simple-",
+			GenerateName: "cpp-simple-",
```

Comment:
I just noticed this here switched from Rus to CPP - was that deliberate?

Reply:
I tried to keep the e2e tests in the order you mentioned in the ticket -  https://buganizer.corp.google.com/issues/309492114#comment1

Reply:
The order doesn't matter at all. You can revert these changes 👍🏻